### PR TITLE
Load .env files when launching from CLI 

### LINKED ISSUE #53: Move credentials.json to .env file
This should align more with best practices for storing sensitive info and dealing with test/live environments.

Need to consider possible backwards compatibility with:
- `credentials.json`
- Command line parameters
- `launchpad.json` config parameters

Also should think about whether there's a global credentials/sensitive info manager class or just one for content sources.

cc @pingevt

### PR DESCRIPTION
**Pull request by claytercek**
resolves #53 

- added support for .env files
- updated documentation to reflect .env support and deprecate crendentials file
- refactored config loading logic to allow loading the env between finding the config and loading the config
- added additional CLI options for specifying env files 

I refactored the `ConfigManager` to just be a couple of exported functions from `config.js`, as now that it's not a singleton, basically all of the methods were just static anyways.

### CODE REVIEW

**Comment by claytercek** [comment]
Tested this a bit more thoroughly, it feels stable to me!

I pulled any recommendation for using `credentials.json` from the docs, replacing it with `.env` examples and instructions, and noted `credentialsPath` as deprecated in the config options.

**Review by benjaminbojko — COMMENTED** [review]
This looks great! Thanks for the PR. Only had one question about `loadConfig()` as defined in `.docs/generate.mjs`. Looks like maybe that's a relic?

Also, should we explicitly mark [`credentials.js`](https://github.com/bluecadet/launchpad/blob/feat/dotenv/packages/content/lib/credentials.js) as deprecated (the class and constructor itself)?

Great addition to the library, and will offer a lot more flexibility.

File: .docs/generate.mjs [hunk 1]
```diff
@@ -2,9 +2,9 @@ import jsdoc2md from 'jsdoc-to-markdown';
 import fs from 'fs-extra';
 import path from 'path';
 import chalk from 'chalk';
-import { ConfigManager, LogManager } from '@bluecadet/launchpad-utils';
+import { loadConfig, LogManager } from '@bluecadet/launchpad-utils';
 
-const config = ConfigManager.getInstance().getConfig();
+const config = loadConfig();
```
[the remark this answers is not in the document]
Comment (claytercek) [reply .docs/generate.mjs]: Oops! Good catch. The `loadConfig` fn briefly existed before it was split into two parts: `findConfig` and `loadConfigFromFile`.

**Review by benjaminbojko — APPROVED** [review]
Great! This looks good to merge to me. @claytercek Do you think we're at a good point to publish to NPM now? Also wondering if this actually constitutes to a minor change for all packages (minus scaffold) instead of just a patch at this point.

**Comment by claytercek** [comment]
Deprecated the Credentials class and all its methods. Unfortunately, there's no way to mark the `credentialsPath` property in the content config options as deprecated with jsDoc, so I just noted it in the description.

**Comment by claytercek** [comment]
Oh yeah, this probably makes more sense as a minor change. I'll update that and merge.

I think once this is merged it would be in a good spot to publish!

**Pull request merged**